### DISCUSSION TITLE
What happens to input-group size modifiers in version 3?

### DISCUSSION THREAD

**Discussion by Mashiane** [saadeghi/daisyui#1920] [Q&A] [answered]
Hi

With input-group being replaced by join in version 3, how does one now handle the input-group-xs|md|lg (size) modifiers?

Thank you.

**Comment by saadeghi** [comment, marked as answer]
You should use the size modifiers for the items: https://play.tailwindcss.com/RLdkRq2ZgX

This provides more consistency because now we can put anything inside a `join` and apply any styles to them. Previously, when we had size modifiers for the input and button and another size modifier for the group (to apply the size again for the input or button) it was confusing and unpredictable that what size modifier would be applied to the item.

**Comment by beeb** [comment]
Hopping on this thread to ask how we can add labels to text inputs like we used to do with input groups? https://daisyui.com/components/input-group/#label-on-both-sides

Is there a component that looks like the `<span>` in input groups in v3?

**Reply by saadeghi** [reply]
See this https://github.com/saadeghi/daisyui/issues/1964

**Reply by beeb** [reply]
Thanks! I used some tailwind classes like suggested for now.
